contracts/tokenomics/maker/src/utils.rs

```diff
@@ -1,13 +1,21 @@
 use crate::error::ContractError;
 use crate::state::{Config, BRIDGES};
-use astroport::asset::{Asset, AssetInfo, PairInfo};
+use astroport::asset::{native_asset_info, Asset, AssetInfo, PairInfo};
 use astroport::maker::ExecuteMsg;
 use astroport::pair::Cw20HookMsg;
 use astroport::querier::query_pair_info;
 use cosmwasm_std::{to_binary, Coin, Deps, Env, StdResult, SubMsg, Uint128, WasmMsg};
 
 pub const BRIDGES_INITIAL_DEPTH: u64 = 0;
+/// The depth of bridges to swap
```

Comment:
Do we need to have these consts to be public?

Reply:
yep